pycyphal/transport/udp/_udp.py

```diff
@@ -64,7 +64,7 @@ class UDPTransport(pycyphal.transport.Transport):
     def __init__(
         self,
         local_ip_address: IPAddress | str,
-        local_node_id: typing.Optional[int] = 0,
+        local_node_id: typing.Optional[int] = None,
```

Comment:
```suggestion
        local_node_id: typing.Optional[int],
```

I originally wrote `None` in the issue but then realized that the default should be absent for consistency with the other transports, and updated the issue accordingly.

Reply:
Yes please.